siliconcompiler/tools/klayout/export.py

```diff
@@ -47,3 +46,10 @@ def setup(chip):
     # Export GDS with timestamps by default.
     chip.set('tool', tool, 'task', task, 'var', 'timestamps', 'true', step=step, index=index, clobber=False)
     chip.set('tool', tool, 'task', task, 'var', 'timestamps', 'Export GDSII with timestamps', field='help')
+
+    chip.set('tool', tool, 'task', task, 'var', 'show_horizontal_resolution', '4096', step=step, index=index, clobber=False)
+    chip.set('tool', tool, 'task', task, 'var', 'show_vertical_resolution', '4096', step=step, index=index, clobber=False)
+
+    # Help
+    chip.set('tool', tool, 'task', task, 'var', 'show_horizontal_resolution', 'Horizontal resolution in pixels', field='help')
```

Comment:
I think you also need to add: https://github.com/siliconcompiler/siliconcompiler/blob/eb8856b41df1bf096ad0fcb45c43c6c3b9fa650d/siliconcompiler/tools/klayout/show.py#L22
Otherwise the hide layers don't get hidden